### PR TITLE
feat(frontend): implement icons for each type of notifications tm-559

### PR DESCRIPTION
**Pull request by Sytnikovakatya**
The icons are shown at the All tab on the Notifications page to distinguish notifications' type
Снимок экрана (6)

### CODE REVIEW

File: apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx [hunk 1]
```diff
@@ -36,6 +39,15 @@ const NotificationListItem: React.FC<Properties> = ({
 
 	const date = getTimeDistanceFormatDate(notification.createdAt);
 
+	const iconName = notificationTypeToIconName[notification.type];
+
+	const isComment = iconName === "follower";
```
Comment (fshabanov) [inline apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx:44]: It is not a comment if the iconName is follower 🤔

**Review by fshabanov — CHANGES_REQUESTED** [review]

**Review by Oleksandra-Masiuk — APPROVED** [review]

**Review by Vfivew — APPROVED** [review]

**Review by Ann-Maksay — APPROVED** [review]

File: apps/frontend/src/pages/notifications/libs/components/notification-list-item/libs/maps/notification-type-to-icon-name.map.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { NotificationType } from "~/libs/enums/enums.js";
+
+const notificationTypeToIconName = {
```
Comment (what1s1ove) [inline apps/frontend/src/pages/notifications/libs/components/notification-list-item/libs/maps/notification-type-to-icon-name.map.ts:3]:
```suggestion
const notificationTypeToIconName: Record<NotificationType , Icon> = {
```

File: apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx [hunk 3]
```diff
@@ -13,14 +13,17 @@ import { useEffect, useInView } from "~/libs/hooks/hooks.js";
 import { type ValueOf } from "~/libs/types/types.js";
 import { type NotificationResponseDto } from "~/modules/user-notifications/user-notifications.js";
 
+import { notificationTypeToIconName } from "./libs/maps/maps.js";
 import styles from "./styles.module.css";
 
 type Properties = {
+	isNotificationFilterAll: boolean;
```
Comment (what1s1ove) [inline apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx:20]:
```suggestion
	hasIcon: boolean;
```

File: apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx [hunk 4]
```diff
@@ -36,6 +39,15 @@ const NotificationListItem: React.FC<Properties> = ({
 
 	const date = getTimeDistanceFormatDate(notification.createdAt);
 
+	const iconName = notificationTypeToIconName[notification.type];
+
+	const isFollower = iconName === "follower";
+
+	const iconClassName = getValidClassNames(
+		styles["icon"],
+		isFollower && styles["icon-follower"],
```
Comment (what1s1ove) [inline apps/frontend/src/pages/notifications/libs/components/notification-list-item/notification-list-item.tsx:48]:
```suggestion
```
it's bad to have specific classes for icons. increase the size of the icon manually then

File: apps/frontend/src/pages/notifications/libs/components/notification-list/notification-list.tsx [hunk 5]
```diff
@@ -16,10 +16,12 @@ import { NotificationListItem } from "../notification-list-item/notification-lis
 import styles from "./styles.module.css";
 
 type Properties = {
+	isNotificationFilterAll: boolean;
```
Comment (what1s1ove) [inline apps/frontend/src/pages/notifications/libs/components/notification-list/notification-list.tsx:19]:
```suggestion
	hasIcon: boolean;
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by YourTeaChan — APPROVED** [review]

**Review by GEOFARL — APPROVED** [review]

**Review by ShadowOfTheSpace — APPROVED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**